spec/Classes.tex

```diff
@@ -314,7 +361,7 @@ \section{Inheritance}
 When inheriting from multiple base classes, only one of the base classes
 may contain fields.  The other classes can only define methods.  Note
 that a class can still be derived from a class that contains fields
-which is itself derived from a class that contains fields.  
+which is itself derived from a class that contains fields.
```

Comment:
Ugh. Would you care to move multiple inheritance to a "future work" section?
This is optional, as it is unrelated to this PR. I could give you a hand with this if you think this should be done.

Reply:
I updated the particular 2 multiple inheritance spots. I'd appreciate help updating any other parts this PR doesn't fix, post-merge.